splash/resources.py

```diff
@@ -201,9 +201,27 @@ def _log_stats(self, request, options):
             "timestamp": int(time.time()),
             "user-agent": (request.getHeader(b"user-agent").decode('utf-8')
                            if request.getHeader(b"user-agent") else None),
-            "args": repr(options)
         }
-        log.msg(json.dumps(msg), system="events")
+        msg.update(self.detailed_options(options))
+        try:
+            msg = json.dumps(msg)
+        except TypeError:
+            # splash-as-proxy options contain byte objects which are not
+            # serializable
+            pass
+        log.msg(msg, system="events")
+
+    def detailed_options(self, options):
+        url = options.get("url")
+        if not url:
+            return options
+        url = six.moves.urllib.parse.urlparse(url)
+        options.update({
+            "netloc": url.netloc,
+            "query": url.query,
+            "path": url.path
+        })
```

Comment:
actually parsing url in kibana could be tricky, i can do some research though. Alternative apprach would be: If you think users could pass anything maybe we could just do some try - except, and try parsing url only if it is valid url?